harness/determined/cli/dev.py

```diff
@@ -257,6 +259,16 @@ def auto_complete_binding(available_calls: List[str], fn_name: str) -> str:
     ]
     if not matches:
         raise errors.CliError(f"no such binding found: {fn_name}")
+    if not skip_partial_match and len(matches) == 1:
```

Comment:
Yes!! I thought this was a nuisance the other day, it'll be nice to automate getting past it. The name "skip_partial_match" isn't ideal IMHO - "skip_auto_picking" (or something else like that) which you use later seems clearer, and consistency would be good.